apps/wallet/src/ui/app/index.tsx

```diff
@@ -49,6 +50,12 @@ const App = () => {
     const isPopup = useAppSelector(
         (state) => state.app.appType === AppType.popup
     );
+
+    const activeNetwork = useAppSelector(({ app }) => app.apiEnv);
```

Comment:
This will be triggered every time App renders, I think we should move it after [this](https://github.com/MystenLabs/sui/blob/main/apps/wallet/src/ui/index.tsx#L33). The store should have the active network there and we can get it with `store.getState().app.apiEnv`